dxr/plugins/rust/__init__.py

```diff
@@ -1075,5 +1075,6 @@ def process_end_external_crates(args, tree):
 
 plugin = Plugin(filters=filters_from_namespace(filters.__dict__),
                 tree_to_index=TreeToIndex,
+                badge_colors={'rust':'burlywood'},
```

Comment:
Btw, to remove the spacing between, we'll need to compress the whitespace out of the markup. Here's what I wrote in layout.html, and it worked:
```
                                  {% for text, color in filter.badges -%}
                                    {%- if color -%}
                                      <span class="badge-label" style="background-color:{{ color }}">{{ text }}</span>
                                    {%- endif -%}
                                  {%- endfor %}
```